homeassistant/components/sensor/neurio_energy.py

```diff
@@ -21,48 +24,133 @@
 CONF_API_SECRET = 'api_secret'
 CONF_SENSOR_ID = 'sensor_id'
 
-DEFAULT_NAME = 'Energy Usage'
+ACTIVE_NAME = 'Energy Usage'
+DAILY_NAME = 'Daily Usage'
```

Comment:
Correct. And no worries, was fun adding support and getting the python neurio component to work with python3.